packages/plugin-video/web/src/video-viewer.jsx

```diff
@@ -89,6 +96,15 @@ class VideoViewer extends Component {
       width,
       height,
       key,
+      ...(disableDownload && {
+        config: {
+          file: {
+            attributes: {
+              controlsList: 'nodownload',
```

Comment:
Let's try this to also disable the context menu
```suggestion
              controlsList: 'nodownload',
              onContextMenu: e => e.preventDefault()
```

Reply:
context menu is disabled by default. merging